client/landing/gutenboarding/onboarding-block/loading-placeholder/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+import './style.scss';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export default function LoadingPlaceholder() {
+	return (
+		<div className="editor__placeholder">
+			<div className="edit-post-layout">
+				<div className="edit-post-header">
+					<div className="edit-post-header-toolbar">
+						<div className="placeholder placeholder-site">Placeholder</div>
+					</div>
+					<div className="edit-post-header__settings">
+						<div className="placeholder placeholder-button">Placeholder</div>
+						<div className="placeholder placeholder-button">Placeholder</div>
+						<div className="placeholder placeholder-button">Placeholder</div>
+					</div>
+				</div>
+				<div className="edit-post-layout__content">
+					<div className="edit-post-visual-editor editor-styles-wrapper">
+						<div className="editor-writing-flow">
+							<div className="editor-post-title">
+								<div className="placeholder placeholder-title wp-block editor-post-title__block">
+									Placeholder
+								</div>
+								<Spinner size={ 100 } />
+								<div className="editor-post-title__text">Your site is being created...</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
```

Comment:
I'd vote to get this merged with `Creating your site…` on a white background. We can update the loading view when we have clear design feedback. Does that seem OK @simison?

Reply:
Yep! Sounds good. The goal is to just get something minimal in so that it's clear in the flow that something is loading. Doesn't matter too much how it looks like, as long as its minimal.

Reply:
Sounds good to me.  With that and with @simison 's suggestion of covering the entire UI, that will look something like this for now: 

![create-site-screen](https://user-images.githubusercontent.com/28742426/73866832-828bcd00-4813-11ea-9fed-595768a60aa4.gif)

After having the Spinner, it feels like something is missing between the last placeholder bar and the "Your site is being created" text.  But it will serve well enough as a temporary style until we something more concrete from design.

Reply:
Hi, yes, can you add this as a needs-design ticket, if you haven't already? And tag the design team?

Reply:
@olesyabrk  sure thing! - I have created a ticket here: #39287